### PR TITLE
temp02.html on new-branch-02

### PR DESCRIPTION
**Pull request by myrewardhua**
we will work on it.

### CODE REVIEW

File: temp01.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 klklklklkljljljlkkljljjljlkjljljllkjlj
 <html>
     <body>
-
+        <p>I added it for new-branch-02</p>
+        my-new-branch-01
```
Comment (chenhualihx) [inline temp01.html]: You should add a tag here.

File: temp02.html [hunk 2]
```diff
@@ -1 +1,7 @@
 sdfsdfsdf in temp02.html
+the followings are added from branch, new-branch-02:
+slfjlsdfjsfjs
+sdlgjslgjsdglsjgl
+sdljksdgjlsd546456
+
```
Comment (chenhualihx) [inline temp02.html]: Why do you have a blank line?

**Review by chenhualihx — CHANGES_REQUESTED** [review]

File: temp01.html (hunk 1 above)
Comment (myrewardhua) [reply temp01.html]: Thanks. I added a <p> tag.

File: temp02.html (hunk 2 above)
Comment (myrewardhua) [reply temp02.html]: Thanks. I removed the blank line

**Review by chenhualihx — APPROVED** [review]

File: temp01.html [hunk 3]
```diff
@@ -1,6 +1,8 @@
 klklklklkljljljlkkljljjljlkjljljllkjlj
 <html>
     <body>
-
```
Comment (chenhualihx) [inline temp01.html:4]: get rid of this blank line.

File: temp02.html [hunk 4]
```diff
@@ -1 +1,8 @@
 sdfsdfsdf in temp02.html
+the followings are added from branch, new-branch-02:
+slfjlsdfjsfjs
+sdlgjslgjsdglsjgl
+sdljksdgjlsd546456
+//added has been done!
+
```
Comment (chenhualihx) [inline temp02.html]: get rid of this blank line.

File: temp01.html (hunk 3 above)
Comment (myrewardhua) [reply temp01.html:4]: I did remove this line.

**Review by myrewardhua — COMMENTED** [review]
good job!

**Review by chenhualihx — APPROVED** [review]

**Pull request merged**